### PR TITLE
OV 6.2 validation

### PR DESCRIPTION
**Pull request by nabhajit-ray**
### Description
[OV 6.2 validation]

### Issues Resolved
[List any issues this PR will resolve. e.g., Fixes #01]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: endpoints-support.md [hunk 1]
```diff
@@ -61,65 +61,65 @@
...
+|<sub>/rest/ethernet-networks/{id}</sub>                                                  | PUT      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |
+|<sub>/rest/ethernet-networks/{id}</sub>                                                  | PATCH    | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:         | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |:white_check_mark:   |
```
Comment (VenkateshRavula) [inline endpoints-support.md:83]: Is this PATCH endpoint marked as ✅ by mistake..?

File: endpoints-support.md [hunk 2]
```diff
@@ -61,65 +61,65 @@
...
+|<sub>/rest/fc-networks/{id}</sub>                                                        | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:         | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |
+|<sub>/rest/fc-networks/{id}</sub>                                                        | PATCH    | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_multiplication_x:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   | :heavy_minus_sign:   |:white_check_mark:   |
```
Comment (VenkateshRavula) [inline endpoints-support.md:92]: Is this PATCH endpoint marked as ✅ by mistake..? Please correct it

File: endpoints-support.md [hunk 3]
```diff
@@ -239,85 +239,85 @@
...
+|<sub>/rest/tasks/{id}</sub>                                                                   | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |
 |<sub>/rest/tasks/{id}</sub>                                                              | GET{Tree View}            | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  | :white_check_mark:  |
```
Comment (VenkateshRavula) [inline endpoints-support.md:303]: Please update the remaining task endpoints for API3000

**Review by VenkateshRavula — APPROVED** [review]

File: endpoints-support.md (hunk 2 above)
Comment (nabhajit-ray) [reply endpoints-support.md]: done

**Review by yuvirani — CHANGES_REQUESTED** [review]
Create EG and associate with range for allocator endpoint

**Review by yuvirani — APPROVED** [review]

**Pull request merged**